src/version.ts

```diff
@@ -37,38 +37,24 @@ export function setPackageJsonVersion(config: Config = getConfig()): void {
 
 async function fetchLatestVersion(
   config: Config = getConfig()
-): Promise<GraphQlQueryResponseData> {
+) {
```

Comment:
Wondering if this is `Promise<OctokitResponse>`

from https://octokit.github.io/types.ts/types/OctokitResponse.html
looking through [@octokit/types](https://octokit.github.io/types.ts/)

Reply:
Yes, that's the correct type for this function. Thankfully octokit is fully typed and this function correctly infers the return type here 👍